collect_app/src/test/java/org/odk/collect/android/entities/EntitiesRepositoryTest.kt

```diff
@@ -428,6 +428,27 @@ abstract class EntitiesRepositoryTest {
         )
     }
 
+    @Test
+    fun `#delete supports list names with dots and dashes`() {
+        val repository = buildSubject()
+
+        val leoville = Entity.New("1", "Léoville Barton 2008")
+
+        repository.save("wines.x", leoville)
+        repository.save("wines-x", leoville)
+
+        repository.delete("1")
```

Comment:
This makes me realize that `delete` should probably take a list as an arg as well, but that's for another time!